DSharpPlus/Net/Models/MembershipScreeningEditModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using DSharpPlus.Entities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DSharpPlus.Net.Models
+{
+    public class MembershipScreeningEditModel : BaseEditModel
+    {
+        /// <summary>
+        /// Sets whether membership screening should be enabled for this guild
+        /// </summary>
+        public bool? Enabled { internal get; set; } = null;
+
+        /// <summary>
+        /// Sets the server description shown in the membership screening form
+        /// </summary>
+        public Optional<string> Description { internal get; set; }
+
+        /// <summary>
+        /// Sets the server rules shown in the membership screening form
+        /// </summary>
+        public DiscordGuildMembershipScreeningField Terms { internal get; set; } = null;
```

Comment:
Doesn't this only edit one field? On the docs it says it takes an array of fields.